### ISSUE TITLE
Dara: Slider overlapping top of content

### ISSUE THREAD

**Issue by thesacredpath** [Automattic/themes#169]
I've not been able to reliably reproduce this in Chrome, Safari, or the Firefox regular version, but can in Firefox Developer Edition nearly every time. I was able to see it once in Chrome.

## Steps to replicate

1. Clear FF Dev cache and then visit the Dara Demo site: [https://darademo.wordpress.com](url)
2. Content is tucked up under the slider. Click on a menu item and then go back to home page and the issue does not occur.
3. Clear browser cache and then reload page and nearly every time the issue will reoccur.

**Screenshot**:
https://cldup.com/fYAKrrrFYm.png

Slack conversation with @crunnells in #themes: https://a8c.slack.com/archives/C029FM1EH/p1529527358000002

## Expected

Content to be properly aligned under slider.

* User report: [https://en.forums.wordpress.com/topic/dara-theme-home-slider-is-over-page-content/](url)
* User's site: [http://closroseraie.wordpress.com/](url)

**Comment by Lochlaer** [comment]
User is reporting this same issue in #1604482-zen.

**Comment by caimin-j** [comment]
Had a user reporting the same issue. 
#1638045-zen

Their screenshot:

![bug](https://user-images.githubusercontent.com/37620825/49821494-dff4cd00-fd7a-11e8-8c4e-9a015a27fe58.png)

Haven't been able to reproduce in Chrome, but it happens in Firefox every time. My screenshot:

<img width="1337" alt="screenshot 2018-12-11 at 19 22 11" src="https://user-images.githubusercontent.com/37620825/49821595-19c5d380-fd7b-11e8-9f83-a96b4f37551c.png">

At least with FF, the issue doesn't seem to be strictly cache related. When resizing the window, I found the issue corrected itself:

<img width="1224" alt="screenshot 2018-12-11 at 19 23 28" src="https://user-images.githubusercontent.com/37620825/49821667-4f6abc80-fd7b-11e8-8ec3-e60976c9e203.png">

Could be something connected with responsive resizing? 

Maybe the cache impacts this sometimes because the theme is trying to resize the image before it's fully loaded (and the true image height/width is available)?

Just some wild guesses thrown in ;)

**Comment by liviopv** [comment]
Also able to reproduce on Firefox.

Doing anything that forces the slider to become "active" (like manually viewing to another slider image) makes the error go away:

Video: http://cld.wthms.co/Tqdp2J

Pinged https://themedevp2.wordpress.com/2018/12/18/master-thread-theme-maintenance-sprint/ to prioritize since this feels like a pretty big bug

**Comment by laurelfulford** [comment]
I was able to recreate this on reloading the theme demo's front page in Firefox; I had less luck recreating when visiting the site fresh, in an existing or incognito window.

**Comment by laurelfulford** [comment]
Tested on https://sebagork.com/, and it's not fixing there, so doubling back to check the fix.

**Comment by laurelfulford** [comment]
Yeesh, 1604482-zen is not an AT site, I don't think I gave it enough time to load the first time 🤦‍♀️ 

It looks like it's fixed there now, too -- I will add a comment to that ZD ticket as well, and close this out.

**Comment by laurelfulford** [comment]
Reopening issue because the fix is causing problems with shorter images in the slideshow:

![image](https://user-images.githubusercontent.com/177561/50667973-e8eb4f80-0f70-11e9-814e-072f4de7e660.png)

The CSS is assuming they're going to be 600px tall and it really, really shouldn't. 

Report and followup: https://en.forums.wordpress.com/topic/featured-content-slider-arrows-fell-out-of-place/

**Comment by laurelfulford** [comment]
An updated fix has been committed in r52799-wpcom-themes.

It's a bit of a nuclear approach -- I've added JavaScript to trigger resize events when the slideshow is complete, and when the images in the slideshow are done loading, to help make sure Firefox has it at the right size. This does mean there can be some overlap while the slideshow is loading, but it will correct itself once all the images are there.

I'm pretty sure this issue is because we changed the theme to use `$(document).ready()` instead of `$(window).load()`, to fix the ads issue in #108. The images aren't always loaded when the slideshow is initiated, causing the overlap.
